### PR TITLE
US002 Hotel Registertration

### PR DESCRIPTION
**Pull request by DuongPhamViet**

### CODE REVIEW

File: B-CLIENT/gotrip/src/property-management/pages/PropertyType/PropertyTypePage.jsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
+import { useEffect, useState } from "react";
+import { withRouter, useParams } from "react-router-dom";
+import PropertyTypeItem from './components/PropertyTypeItem';
+import { getPropertyTypeByCategory } from "../../actions/property";
+import { useDispatch, useSelector } from "react-redux";
+import ConfirmationModal from "../../../shared/components/forms/ConfirmationModal";
+import ReactCountryFlag from "react-country-flag";
+
+const PropertyTypePage = () => {
+    const [idSelected, setIdSelected] = useState('');
```
Comment (dnhieu1992) [inline B-CLIENT/gotrip/src/property-management/pages/PropertyType/PropertyTypePage.jsx:10]: Using redux to manage the state

File: B-CLIENT/gotrip/src/property-management/pages/PropertyType/PropertyTypePage.jsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    const [numberOfItemShow, setNumberOfItemShow] = useState(6);
+    const [showModal, setShowModal] = useState(false)
+
```
Comment (dnhieu1992) [inline B-CLIENT/gotrip/src/property-management/pages/PropertyType/PropertyTypePage.jsx:13]: Using redux to manage the states: idSelected, numberOfItemShow, showModal

File: B-CLIENT/gotrip/src/property-management/pages/PropertyType/PropertyTypePage.jsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                    }
+                </div>
+
+                <button className="missing-property" onClick={openModal}>
+                    <span className="missing-property__icon"><i className="fa fa-question"></i></span> I don't see my property type on the list
+                </button>
+
+                <div className="property-button">
+                    <hr />
+                    <button className="property-button__back"><i className="fa fa-chevron-left"></i></button>
```
Comment (dnhieu1992) [inline B-CLIENT/gotrip/src/property-management/pages/PropertyType/PropertyTypePage.jsx:102]: Import icon and use the same GridHeader.jsx

File: B-CLIENT/gotrip/src/property-management/pages/PropertyType/PropertyTypePage.jsx (hunk 2 above)
Comment (DuongPhamViet) [reply B-CLIENT/gotrip/src/property-management/pages/PropertyType/PropertyTypePage.jsx]: done

File: B-CLIENT/gotrip/src/property-management/pages/PropertyType/PropertyTypePage.jsx (hunk 3 above)
Comment (DuongPhamViet) [reply B-CLIENT/gotrip/src/property-management/pages/PropertyType/PropertyTypePage.jsx]: done

**Pull request merged**